.gitignore

```diff
@@ -87,3 +87,4 @@ htmlcov/
 .coverage
 # ansible-test coverage results
 test/units/.coverage.*
+.venv/
```

Comment:
This change should be made in a separate PR since it's not directly related to the rest of the changes here.